lib/dialects/sqlite/connection-manager.js

```diff
@@ -55,6 +55,15 @@ ConnectionManager.prototype.getConnection = function(options) {
       // explicitly disallowed. It's still opt-in per relation
       connection.run('PRAGMA FOREIGN_KEYS=ON');
     }
+
+    // PRAGMA commands to support sqlcipher database encryption
+    if (self.sequelize.options.key) {
+      connection.run('PRAGMA KEY = \'' + self.sequelize.options.key + '\'');
+    }
```

Comment:
Would you like a object in the opts of:

sqlcipher: {
  key: '<key>',
  cipher: '<cipher'
}